### DISCUSSION TITLE
Renovate bot run from pipeline end

### DISCUSSION THREAD

**Discussion by nickkotchauri** [renovatebot/renovate#17167] [Request Help]
Hello, can i config renovatebot run check after pipeline i update my nuget packs

**Comment by PhilipAbed** [comment]
Nope, if you are already using a pipeline, and you want to trigger it per pipeline run, isn't it better if you use self hosted instead? then you can run it however you wish.
RenovateBot runs on a server, and it triggers every X hours or after certain github webhooks
